e2e/bitrise.yml

```diff
@@ -56,36 +57,53 @@ workflows:
     - TEST_APP_BRANCH: failing-ui-test
     - BITRISE_PROJECT_PATH: ios-simple-objc/ios-simple-objc.xcodeproj
     - BITRISE_SCHEME: ios-simple-objc
-    - XCODE_SIMULATOR_DEVICE: iPhone 8 Plus
+    - XCODE_SIMULATOR_DEVICE: iPhone 8
     - XCODE_SIMULATOR_OS_VERSION: "latest"
     - XCODE_SIMULATOR_PLATFORM: iOS Simulator
+    - XCODE_OUTPUT_TOOL: xcodebuild
+    - RETRY_ON_FAILURE: "yes"
+    - EXPECT_TEST_FAILURE: "true"
     after_run:
     - _run
     - _check_outputs
     - _check_exported_artifacts
 
-  test_tvos:
+  test_parallel:
     envs:
-    - TEST_APP_URL: https://github.com/bitrise-io/sample-apps-tvos-swift.git
-    - TEST_APP_BRANCH: master
-    - BITRISE_PROJECT_PATH: NPO Live.xcworkspace
-    - BITRISE_SCHEME: NPO Live
-    - XCODE_SIMULATOR_DEVICE: Apple TV
+    - TEST_APP_URL: https://github.com/bitrise-io/sample-swift-project-with-parallel-ui-test.git
+    - TEST_APP_BRANCH: STEP-1054-test-plan
+    - BITRISE_PROJECT_PATH: BullsEye.xcworkspace
+    - BITRISE_SCHEME: BullsEye
+    - XCODE_SIMULATOR_DEVICE: iPad Air (4th generation)
```

Comment:
Seems like it is not available with Xcode 10.3: https://app.bitrise.io/build/b48b00c0a231e6b7#?tab=log